satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -626,13 +626,23 @@ model certRecord (
 
 create certRecord ( )
 delete certRecord ( where certRecord.id = ? )
-update certRecord ( where certRecord.id = ? )
 
 read one (
 	select certRecord
 	where  certRecord.id = ?
 )
 
+read one (
+	select certRecord
+	where  certRecord.publickey = ?
+)
+
+read all (
+	select certRecord
+	where  certRecord.id = ?
+	orderby desc certRecord.update_at
+)
```

Comment:
You should add an index on id and update_at

Reply:
updated as suggested